app/components/Feedback/feedback.css

```diff
@@ -1,23 +1,49 @@
-.feedback .composite-button {
-  width: fit-content;
+.feedback {
+  display: flex;
+  align-items: center;
+}
+
+.thanks {
+  transition: opacity 200ms ease-in-out;
+  margin-left: 0.5rem;
+  pointer-events: none;
+}
+.show {
+  opacity: 1;
+}
+
+.hide {
+  opacity: 0;
 }
 
 .feedback-form {
-  position: relative;
-  z-index: 2;
-  max-width: 384px;
-  padding: var(--spacing-32);
-  margin-top: var(--spacing-8);
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 50px;
+  left: calc(13.333vw + 85px);
+  width: 384px;
+  padding: 30px;
+  border-radius: 6px;
 }
 
 /* Comment to Nemo when he reviews: why didn't you just do this the way you did the settings buttons? */
```

Comment:
check `git blame`. I didn't do this :D Feel free to make things more consistent everywhere

Reply:
Heh, I think I was referencing something else which I now forget

Reply:
But Myles could you use a variable instead of padding: 30, that way we'll always stay using my system of multiples of 8. That said I imagine it's better to just set the height in this case. But yeah just change it to a var and we're good for now